main/main.go

```diff
@@ -2,16 +2,52 @@ package main
 
 import (
 	"fmt"
+	"os"
+	"strconv"
+	"time"
 
-	"github.com/thermo-smart/TemperatureSensorController"
+	"github.com/gaeshi-gaeshi/thermo-smart/HeatersController"
+	"github.com/gaeshi-gaeshi/thermo-smart/TemperatureSensorController"
 )
 
 func main() {
-	var temp, error = TemperatureSensorController.ReadTemperature()
+	if os.Args[1] == "--reset" {
```

Comment:
os.Args[1] - could be assigned to meanigful variable

Reply:
And what name would you propose? It is meaningless to name it "firstArgument".